UnleashClient/constants.py

```diff
@@ -6,6 +6,9 @@
 
 # =Unleash=
 APPLICATION_HEADERS = {"Content-Type": "application/json"}
+DISABLED_VARIATION = {
```

Comment:
should also have `enabled` set to false.

Reply:
Tied to comment above re: `default_value`/`fallback_value`: In cases that variations should be disabled (i.e. no variants, excepts, etc), I was evaluating the feature flag to determine if something was enabled/disabled.  The main case would be if someone set `default_value` = True.

If we don't want to allow tweaking the `is_enabled()` value, I can default `enabled` here.

Reply:
Fixed as part of ^